Containerfile.stream9

```diff
@@ -2,7 +2,7 @@ FROM quay.io/centos/centos:stream9
 
 MAINTAINER CRC team
 
-RUN dnf --enablerepo=crb install -y libvirt-devel curl git gcc make golang diffutils
+RUN dnf --enablerepo=crb install -y libvirt-devel git gcc make golang diffutils
```

Comment:
This could be squashed in the previous commit

Reply:
Done